composer.json

```diff
@@ -20,11 +20,12 @@
     ],
     "require": {
         "php": "^5.5 || ^7.0",
-        "zendframework/zend-modulemanager": "^2.5",
-        "zendframework/zend-loader": "^2.5",
-        "zendframework/zend-eventmanager": "^2.5 || ^3.0",
-        "zendframework/zend-mvc": "^2.5 || ^3.0",
-        "zendframework/zend-navigation": "^2.5"
+        "laminas/laminas-modulemanager": "^2.5",
+        "laminas/laminas-loader": "^2.5",
+        "laminas/laminas-eventmanager": "^2.5 || ^3.0",
+        "laminas/laminas-mvc": "^2.5 || ^3.0",
+        "laminas/laminas-navigation": "^2.5",
+        "laminas/laminas-dependency-plugin": "^1.0"
```

Comment:
this can be removed I think as `laminas/laminas-dependency-plugin` mostly used for project, not module.